specs.md

```diff
@@ -145,14 +144,13 @@ ID | Title | Description | Associated Competition Rule(s)
 --- | --- | --- | ---
 PDM-16 | Selective e-fuse enabling | The PDM must only enable the following e-fuse outputs in the AIR-Open state: AUX 1, AUX 2, Energy Meter, CAN, AIR SHDN
 PDM-15 | Entering the AIR-Open state | The PDM must enter the AIR-Open state after the init state is complete.
-PDM-17 | Exiting the AIR-Open state | The PDM must enter the AIR-Closed state when the BMS closes the contactors.
+PDM-17 | Exiting the AIR-Open state | The PDM must enter the AIR-Closed state when the BMS closes the AIR+ and AIR-.
 
 ### PDM AIR-Closed State <a name="PDM_AIR_CLOSED"></a>
 ID | Title | Description | Associated Competition Rule(s)
 --- | --- | --- | ---
 PDM-19 | Selective e-fuse enabling | The PDM must enable all e-fuse outputs in the AIR-Closed state, except for the inverters, which depend on the following:. <br/> <br/> If the start switch is off AND the car is travelling slower than 10km/h, both inverters must be disabled. Otherwise, both inverters must be enabled, to protect against BEMF.
-PDM-18 | Entering the AIR-Closed state | The PDM must enter the AIR-Closed state when the BMS closes the contactors.
-PDM-20 | Exiting the AIR-Closed state | The PDM must enter the AIR-Open state when the BMS opens the contactors.
+PDM-20 | Exiting the AIR-Closed state | The PDM must enter the AIR-Open state when the BMS opens the AIR+ or AIR-.
```

Comment:
I believe the intention here is that if _either_ AIR is open, we should go to AIR open state. Not _both_ contactors.